client/src/shoe-row.js

```diff
@@ -0,0 +1,92 @@
+import React, { PureComponent } from 'react';
+import { shape, bool } from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+import Avatar from '@material-ui/core/Avatar';
+import AssessmentIcon from '@material-ui/icons/Assessment';
+import IconButton from '@material-ui/core/IconButton';
+import deepOrange from '@material-ui/core/colors/deepOrange';
+import map from 'lodash/map';
+import isEqual from 'lodash/isEqual';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+@withStyles({
+  orangeAvatar: {
+    color: '#fff',
+    backgroundColor: deepOrange[500],
+  },
+})
+export default class ShoeRow extends PureComponent {
+  static propTypes = {
+    data: shape(Object),
```

Comment:
`shape` is not really useful in this case.

Reply:
PropType.object is forbidden as per esLint rule.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md

Reply:
Oh, ok I didn't know you were using these ESLint rules. But `PropTypes.shape(Object)` does the same as `PropTypes.object` and is exactly what this rule is supposed to prevent: `vague prop types`. 
So in this case I think you should be more explicit on your intention and disable the rule inline:
```javascript
data: PropTypes.object, // eslint-disable-line react/forbid-prop-types
```